src/overview/reducer.js

```diff
@@ -171,6 +249,24 @@ const changeHasBookmark = (state, index) => {
     return { ...state, searchResult }
 }
 
+const setFilterPopup = (state, name) => {
+    let filterPopup = state.filterPopup
+
+    if (!filterPopup.length) {
```

Comment:
Another spot that could some comments to clear up what conditions are being checked and why.